### DISCUSSION TITLE
Week 5

### DISCUSSION THREAD

**Discussion by jakegros** [USU-ITLS-x265-23-summer/class-repo#114] [General]
<h1>Week 5 Reflection Thoughts</h1>
<h2>Learning Material</h2>
Working my way through the freeCodeCamp JS I have barely begun to get a better hold on it.  The disjointed bite-sized lessons without some visual output and no connected "project" had been confusing and unfulfilling, thus hard to stick with.  However, going through it systematically has started to pay off and lay a better foundation for me. Persevering through it, I can sometimes see how pieces go together.  I am still convinced that there needs to be some sort of practical application to see what the code changes are doing.  I definitely want more practice and application with it.  This is what I have been spending the bulk of my time with.  
There was one lesson I got stuck on and found that this section of fCC has help videos.

Flexbox Froggy was instrumental in helping me understand CSS. Flexbox Froggy lets you literally see what the output is from your code. It was fun and easy to stick with.  My son (he's three) even enjoyed taking on the challenges.  It was quite fun to see his take on it. 

<h2>Project Updates</h2>
The two project ideas I would implement are probably going to be a website or a game. 
I am sure it will be using HTML for the content and CSS to stylize the page.  Javascript will be crucial for a game and I expect would allow for a more dynamic website. It would allow for more logic with variables and inputs. Additionally, I'm exploring GitHub pages to start testing out some ideas and get some ideas rolling.

**Comment by metasean** [comment]
Learning HTML, CSS, and JavaScript at the same time has both it's pros and cons.  The fundamentals HTML/CSS and the fundamentals of JavaScript are really independent of each other, but once you start using the JavaScript in conjunction with the HTML/CSS, it should start making a lot more sense!

> Additionally, I'm exploring GitHub page

💟
